testsuite/manualtests/test_stdp_symmetric.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
+#
+# test_stdp_symmetric.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+# Begin Documentation
+# Name: testsuite::test_stdp_symmetric - script to test stdp_symmetric_ model
+# implementing a symmetric stdp learning rule
+# Two neurons, which fire poisson like, are connected by a
+# stdp_symmetric_synapse model.
+#
+# author: Ankur Sinha
+# date: January 2016
+
+import numpy as np
+import nest
+import math
+import matplotlib
+matplotlib.use("Agg")
+from matplotlib import pyplot as plt
+
+output_graph_name = 'symmetric_stdp.png'
+
+plus_deltaWs = []
+minus_deltaWs = []
+minus_postWs = []
+plus_postWs = []
+minus_deltaTs = [x for x in range(-100, 0)]
+plus_deltaTs = [x for x in range(1, 101)]
+weight_pre = 60.
+
+# Run various tests over different deltaTs
+for deltaT in minus_deltaTs:
+    nest.ResetKernel()
+    nest.SetKernelStatus(
+        {
+            'resolution': 0.01,
+        }
+    )
+    nest.set_verbosity('M_INFO')
+    #################################################################
+    # To get the kplus value up
+    first_spike_set = [(10. + 2. * math.fabs(deltaT) * i)
+                       for i in range(0, 1000)]
+    later_spike_set = [pre_spike + math.fabs(deltaT)
+                       for pre_spike in first_spike_set]
+    # an extra spike to equal the depression in second case
+    # later_spike_set.append(later_spike_set[-1] + math.fabs(deltaT))
+    pre_generator = nest.Create('spike_generator', 1,
+                                {'spike_times': later_spike_set}
+                                )
+    post_generator = nest.Create('spike_generator', 1,
+                                 {'spike_times': first_spike_set}
+                                 )
+    pre_neuron = nest.Create('parrot_neuron')
+    post_neuron = nest.Create('parrot_neuron')
+
+    nest.Connect(pre_generator, pre_neuron)
+    nest.Connect(post_generator, post_neuron)
+
+    # set up the synapse
+    syn_spec_synapse = {'weight': weight_pre, 'Wmax': 100.,
+                        'kappa': 0.1, 'eta': 0.001,
```

Comment:
kappa was replaced by alpha after the review. The manual test file hasn't been updated since it'll be replaced by an automated test (I'm working on the latter)